extensions/amp-form/0.1/amp-form.js

```diff
@@ -526,8 +527,11 @@ export class AmpForm {
         }).then(() => {
           request = this.requestForFormFetch(
               dev().assertString(this.xhrAction_), this.method_);
-          setupInit(request.fetchOpt);
-          setupAMPCors(this.win_, request.xhrUrl, request.fetchOpt);
+          request.fetchOpt = setupInit(request.fetchOpt);
```

Comment:
So the `setupXxx` function actually has a side effect (it mutates the passed-in argument), so maybe it be left in the previous state where the return value is simply ignored... When I see a function that takes an input and returns an ouput, I often assume that it's side effect-free.

Reply:
Hmmm problem is we have to use `setupInit` this way, because it doesn't mutate passed-in parameters.

https://github.com/ampproject/amphtml/blob/master/src/utils/xhr-utils.js#L250

I used others this way for consistency, but can definitely change them if that's better.

Reply:
Yea, the old code was just wrong. Thanks for fixing this.